aspnetcore/signalr/hub-filters.md

```diff
@@ -128,12 +128,12 @@ public class LanguageFilter : IHubFilter
     public async ValueTask<object> InvokeMethodAsync(HubInvocationContext invocationContext, Func<HubInvocationContext, ValueTask<object>> next)
     {
         var languageFilter = (LanguageFilterAttribute)Attribute.GetCustomAttribute(invocationContext.HubMethod, typeof(LanguageFilterAttribute));
-        if (invocationContext.HubMethodArguments.Count < languageFilter.FilterArgument &&
+        if (invocationContext.HubMethodArguments.Count > languageFilter.FilterArgument &&
             invocationContext.HubMethodArguments[languageFilter.FilterArgument] is string str)
         {
             foreach (var bannedPhrase in bannedPhrases)
             {
-                str.Replace(bannedPhrase, "***");
+                str = str.Replace(bannedPhrase, "***");
```

Comment:
👍

Reply:
The example doesn't work. I think we need to put some bannedPhrases in there :wink:. We could include phrases like "async void" and ".Result" 😄 

I liked the Synchronize example, but I agree this is better.

Reply:
That's what the comment about "populate this from a file" is for.

Reply:
I know that, but can't we have a bit of fun?

Reply:
We should do validation, it's pretty common. DataAnnoations based filter 😄

Reply:
You just don't like being censored!

Reply:
We could definitely use this in JabbR2.0. the banned list should just be really short.